partner_event/views/res_partner_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Tecnativa S.L. - Vicent Cubells
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+
+<record id="act_partner_registration" model="ir.actions.act_window">
+    <field name="res_model">event.registration</field>
+    <field name="view_type">form</field>
+    <field name="name">Registrations</field>
+    <field name="view_mode">tree,form,calendar,graph</field>
+    <field name="domain">[("attendee_partner_id", "child_of", active_ids)]</field>
+    <field name="context">{'search_default_attendee_partner_id': active_id, 'default_attendee_partner_id': active_id, 'search_default_expected': True}</field>
+</record>
+
+<record model="ir.ui.view" id="view_partner_form_registrations">
+    <field name="name">Partner Form with registrations</field>
+    <field name="model">res.partner</field>
+    <field name="inherit_id" ref="base.view_partner_form"/>
+    <field name="arch" type="xml">
+        <div name="button_box" position="inside">
+            <button name="%(partner_event.act_partner_registration)d"
+                    type="action"
+                    class="oe_stat_button oe_inline"
+                    help="Count of events with confirmed registrations."
+                    icon="fa-futbol-o">
+                <field name="event_count"
```

Comment:
[This exists upstream](https://github.com/odoo/odoo/blob/2cd730998bd1630a4386607186290c7257df39cd/addons/event/views/res_partner_views.xml#L14). Use `registration_count` instead.

Also the string should be different to avoid this duplication:

![captura de pantalla de 2018-05-14 09-40-06](https://user-images.githubusercontent.com/973709/39987000-54ae9e30-575b-11e8-9fc7-c0b0c36f6da6.png)

Reply:
Maybe we could even get rid of that stat button.

Reply:
Forget about it. I thougth what Odoo had added upstream was the attendes one. Rename to 'Registrations' or 'Attendances'